indexer/lib/fwss-handlers.js

```diff
@@ -64,18 +65,36 @@ export async function handleFWSSDataSetCreated(
 /**
  * Handle Filecoin Warm Storage Service service termination
  *
- * @param {{ DB: D1Database }} env
+ * @param {{
+ *   DB: D1Database
+ *   DEFAULT_LOCKUP_PERIOD_DAYS: number
+ *   FILECOIN_GENESIS_BLOCK_TIMESTAMP_MS: number
+ * }} env
  * @param {any} payload
  * @throws {Error}
  */
 export async function handleFWSSServiceTerminated(env, payload) {
+  // Convert block_number (epoch) to Unix timestamp (in milliseconds)
+  const epochTimestampMs = epochToTimestampMs(
+    payload.block_number,
+    Number(env.FILECOIN_GENESIS_BLOCK_TIMESTAMP_MS),
+  )
+
+  // Calculate lockup unlock timestamp based on the epoch timestamp (in milliseconds)
+  const lockupUnlocksAtMs =
+    epochTimestampMs +
+    Number(env.DEFAULT_LOCKUP_PERIOD_DAYS) * 24 * 60 * 60 * 1000
```

Comment:
are we concerned that `DEFAULT_LOCKUP_PERIOD_DAYS` can get out of sync with the contract? If so, can we read it from it instead of from env?

Reply:
IIRC, we agreed with @pyropy to use the constant so that we can ship this sooner. `DEFAULT_LOCKUP_PERIOD_DAYS` is unlikely to change often.

How about creating a follow-up tech-debt issue for addressing this later? If we feel strongly about this, we can add the new issue to the GA launch backlog.